packages/sql/src/RequestManager.ts

```diff
@@ -26,14 +26,17 @@ export class RequestManager {
   private _retryTimeoutId: number = UNKNOWN;
   private _fetching: boolean = false;
   private _scheduleDebounce: number = UNKNOWN;
+  private _maxRetries: number = MAX_RETRIES;
 
-  constructor(credentials: Credentials) {
+  constructor(credentials: Credentials, {maxRetries}: {maxRetries?: number} = {}) {
     const { username, apiKey, server } = credentials;
 
     this._username = username;
     this._apiKey = apiKey;
     this._server = server.replace('{user}', username);
     this._queue = [];
+
+    if (maxRetries || maxRetries === 0) { this._maxRetries = maxRetries; }
```

Comment:
I don't get the `(maxRetries === 0)` comparison. Could you explain it a bit?

Reply:
To let you set it to 0 in case we define another default value. The falsy value of `maxRetries` would not pass the test. Anyway, I've change it to a restricted test.